wp-content/plugins/core/src/Query/Search.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Nothing wrong with handling this this way. Another way we've handled this in the past is using JS to prevent form submit if the input is empty, but I think that's also been on a more project by project basis.